samples/gltf_viewer.cpp

```diff
@@ -549,8 +549,9 @@ static bool checkAsset(const utils::Path& filename) {
 
     // Parse the glTF file and create Filament entities.
```

Comment:
looks like this comment is wrong too, right? It's not actually creating Filament entities here.

Reply:
Yeah, it seems like this function used to behave like what the comment says and was split into pieces later. Let me correct the comment.